components/compliance-service/api/tests/07_suggestions_control_tags_spec.rb

```diff
@@ -0,0 +1,136 @@
+##### GRPC SETUP #####
+require 'api/reporting/reporting_pb'
+require 'api/reporting/reporting_services_pb'
+
+describe File.basename(__FILE__) do
+  Reporting = Chef::Automate::Domain::Compliance::Api::Reporting unless defined?(Reporting)
+
+  def reporting
+    Reporting::ReportingService;
+  end
+
+  it "errors" do
+    assert_grpc_error("'control_tag' filter is required for 'control_tag_value' suggestions", 2) do
+      GRPC reporting, :list_suggestions, Reporting::SuggestionRequest.new(
+        type: 'control_tag_value',
+        filters: []
+      )
+    end
+  end
+
+  it "works" do
```

Comment:
I used the `it "works"` pattern because a lot of existing tests couldn't be broken up easily as later tests depended on data created in earlier tests. For a brand new test file it'd be better for readability and debugging to break these into individual tests that could fail independently.

Reply:
Thanks Pete, I wrapped an `it` block around each test in a new commit.